src/components/Checkout/OrderTotals.vue

```diff
@@ -1,6 +1,10 @@
 <template>
-	<div class="order-totals row">
-		<div v-if="showKivaCredit" class="kiva-credit columns small-12">
+	<div class="order-totals">
+		<div class="order-total" v-if="showPromoCreditTotal">
+			<strong>Order Total: <span class="total-value">{{ itemTotal }}</span></strong>
+		</div>
```

Comment:
This a is a new block we could potentially show for standard checkout, however it's Promo only for now.

Reply:
👏